examples/call_multiple_tools_directly_with_auth.py

```diff
@@ -0,0 +1,176 @@
+"""Example script demonstrating how to call multiple tools directly with authentication.
```

Comment:
Same example as `call_multiple_tools_with_llm.py` but instead it only calls the tools directly.